backends/edxorg.py

```diff
@@ -16,6 +14,7 @@ class EdxOrgOAuth2(BaseOAuth2):
     name = 'edxorg'
     ID_KEY = 'edx_id'
     REQUEST_TOKEN_URL = None
+    EDXORG_BASE_URL = getattr(settings, 'EDXORG_BASE_URL', 'https://courses.edx.org/')
```

Comment:
It seems like `settings.EDXORG_BASE_URL` should always exist here. Therefore the third argument here would never be used. Also, it seems like if `https://courses.edx.org/` was meant to be a default value, this should go in `settings.py` as `get_var('EDXORG_BASE_URL', 'https://courses.edx.org/')`